util/check_translation.py

```diff
@@ -212,12 +212,26 @@ def translate_timeline(line, trans):
     return line
 
 
-def print_timeline(locale, timeline_file, trans):
+def print_timeline(locale, timeline_file, trans, missing_filter):
```

Comment:
These changes are nice, but in the future can you separate out changes to utilities and updating translations into separate pull requests?